count/utils.go

```diff
@@ -0,0 +1,43 @@
+package count
+
+import (
+	"errors"
+	"math"
+
+	"golang.org/x/exp/constraints"
+)
+
+func Min[T constraints.Ordered](a, b T) T {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func SuggestNumBuckets(relativeError float64) (int32, error) {
+	if relativeError <= 0 {
+		return 0, errors.New("relative error must be greater than 0.0")
+	}
+	return int32(math.Ceil(math.Exp(1.0) / relativeError)), nil
+}
+
+func SuggestNumHashes(confidence float64) (int8, error) {
+	if confidence < 0 || confidence > 1.0 {
+		return 0, errors.New("confidence must be between 0 and 1.0 (inclusive)")
+	}
+	return Min(int8(math.Ceil(math.Log(1.0/(1.0-confidence)))), int8(math.MaxInt8)), nil
+}
+
+func checkHeaderValidity(preamble, serVer, familyID, flagsByte byte) error {
+	return nil
+}
+
+const (
+	PreambleLongsShort = 2
+	SerialVersion1     = 1
+	FamilyId           = 18
```

Comment:
Can you move that familyId in https://github.com/apache/datasketches-go/blob/04e51ea1586a0a87371baee010dcaeb8a3f1a62b/internal/family.go ? thanks